src/twisted/conch/scripts/ckeygen.py

```diff
@@ -96,12 +105,13 @@ def handleError():
     raise
 
 
-
+@_support('rsa')
```

Comment:
While I'm here I should note that the decorator being added to these methods changes their function signature. The decorator should probably use [`functools.wraps`](https://docs.python.org/2/library/functools.html#functools.wraps) to fix that up.